src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const resultingObject = {
+    result: 0,
+
+    add(a, b = 0) {
+      return a + b;
+    },
+
+    subtract(a, b = 0) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      return a / b;
```

Comment:
You need to add a check by 0, because according to the rules of mathematics, you cannot divide by 0